packages/components/src/Persona.purs

```diff
@@ -559,3 +569,13 @@ getPayments uuid token =
              , state: s
              , discount: x.discount
              }
+
+type Forbidden = PersonaError
+  ( forbidden :: { description :: String } )
+
+searchUsers :: String -> Credentials -> Aff (Array User)
+searchUsers query auth = do
+  callApi adminApi "adminSearchGet" [ unsafeToForeign query ]
+    { authorization: oauthToken auth.token
+    , authUser: auth.uuid
```

Comment:
This will end up being the wrong uuid if there are users searched and set to local storage.

Don't know if I like this strategy of using the `uuid` of local storage to serve different purposes. It was designed to be one part of the authentication tuple of (user uuid, auth token), but now it can be something very different too. I suspect this might yield weird bugs in the future.

Didn't still think this all the way through, but if local storage is required, could we leave the original uuid and token untouched and add an optional `activeUserUuid` to indicate if we want to impersonate a customer?

Reply:
I made first page load to always reset the uuid to the adminUuid if it is set, so that acts as a fail safe to reset the uuid to the one of the logged in user.

A pattern I'm seeing here is that uuid and token are common parameters that are passed with just about every call and uuid is the one that may need to vary based on search, but never between any sequence of calls. I was already thinking of bringing `ReaderT` to it but my changes seemed to be large enough even as is and I bypassed it by using `LocalStorage` as an ad hoc State monad. But I guess it's better to add `ReaderT` to this after all, even if it would take a bit longer to implement.